src/common/regions/types.ts

```diff
@@ -166,8 +166,9 @@ export class USA extends Region {
   abbreviation = 'USA';
 
   get relativeUrl(): string {
-    throw new Error('Method not implemented.');
+    return '';
```

Comment:
this^ allows us to do [this](https://github.com/covid-projections/covid-projections/pull/5571/files#diff-58ca9c7001c0a6e58af46f7c23cbc1f92052e4a6c62a9eace7b052cadfaec7bbR51), and doesnt seem to break anything in the process?

Reply:
Seems reasonable!